frontstage_api/resources/surveys/get_surveys_list.py

```diff
@@ -25,21 +25,37 @@ class GetSurveysList(Resource):
     @auth.login_required
     @surveys_api.expect(parser)
     def get():
-        party_id = request.args['party_id']
+        respondent_id = request.args['party_id']
         survey_list = request.args['list']
-        logger.info('Attempting to retrieve surveys list', party_id=party_id, survey_list=survey_list)
+        logger.info('Retrieving surveys list', respondent_id=respondent_id, survey_list=survey_list)
+
+        cases = case_controller.get_case_by_party_id(respondent_id, case_events=True)
 
-        cases = case_controller.get_case_by_party_id(party_id, case_events=True)
         # Filter out the cases relevant to the request
         if survey_list == 'todo':
-            filtered_cases = [case for case in cases if case.get('caseGroup', {}).get('caseGroupStatus') not in ['COMPLETE', 'COMPLETEDBYPHONE']]
+            filtered_cases = [case
+                              for case in cases
```

Comment:
Odd formatting? the original was easier to read.

Reply:
:+1: I added this because it made it easier for me to understand when i was doing it. I agree it looks cleaner in the previous way though

Reply:
If we don't do this we go over line length limits